connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConnector.java

```diff
@@ -138,7 +138,9 @@ private void pause() {
 
                 case STARTED:
                     connector.stop();
-                    // fall through
```

Comment:
I know a few other places where we use fall throughs. I didn't find a way to completely disable a specific rule.

Should we use `suppressFBwarnings` annotation in the code instead of adding the suppressions in a whitelist file (`findbugs-exclude.xml`)? I was thinking that keeping suppressions in a file in sync with the code might be a burden. Having the annotations in the code might be easier to maintain. What do you think?